linera-service/src/faucet.rs

```diff
@@ -46,6 +48,19 @@ impl IntoResponse for FaucetError {
     }
 }
 
+/// The result of a successful `claim` mutation.
+#[derive(SimpleObject)]
+pub struct ClaimOutcome {
+    /// The ID of the message that created the new chain.
+    pub message_id: MessageId,
+    /// The ID of the new chain.
+    pub chain_id: ChainId,
+    /// The hash of the parent chain's certificate containing the `OpenChain` operation.
+    pub open_chain_certificate_hash: CryptoHash,
+    /// The hash of the parent chain's certificate containing the `Transfer` operation.
+    pub transfer_certificate_hash: CryptoHash,
```

Comment:
@afck One thing we could do is use only one block for both operations!

Reply:
Done in https://github.com/linera-io/linera-protocol/pull/1201.